mobile-app/app/screens/AppNavigator/screens/Auctions/screens/AuctionDetailScreen.tsx

```diff
@@ -1,90 +1,106 @@
 import { getPrecisedTokenValue } from "../helpers/precision-token-value";
+import { useTokenPrice } from "../../Portfolio/hooks/TokenPrice";
+import { AuctionVaultDetails } from "../components/AuctionVaultDetails";
+import { AuctionsParamList } from "../AuctionNavigator";
 
 type BatchDetailScreenProps = StackScreenProps<
   AuctionsParamList,
   "AuctionDetailScreen"
 >;
 
-enum TabKey {
-  BidHistory = "BID_HISTORY",
-  Collaterals = "COLLATERALS",
-  AuctionDetails = "AUCTION_DETAILS",
-}
-
 export function AuctionDetailScreen(
   props: BatchDetailScreenProps
 ): JSX.Element {
-  const navigation = useNavigation<NavigationProp<AuctionsParamList>>();
   const { batch: batchFromParam, vault } = props.route.params;
   const [batch, setBatch] = useState<LoanVaultLiquidationBatch>(batchFromParam);
+
+  const { address } = useWalletContext();
+  const { isLight } = useThemeContext();
+  const navigation = useNavigation<NavigationProp<AuctionsParamList>>();
   const client = useWhaleApiClient();
   const dispatch = useAppDispatch();
-  const { getAuctionsUrl } = useDeFiScanContext();
-  const [activeTab, setActiveTab] = useState<string>(TabKey.BidHistory);
-  const { minNextBidInToken, totalCollateralsValueInUSD, minNextBidInUSD } =
-    useAuctionBidValue(batch, vault.liquidationPenalty);
+
+  const { auctions } = useSelector((state: RootState) => state.auctions);
   const blockCount = useSelector((state: RootState) => state.block.count) ?? 0;
-  const { blocksRemaining } = useAuctionTime(
-    vault.liquidationHeight,
-    blockCount
+  const bidHistory = useSelector(
+    (state: RootState) => state.auctions.bidHistory
   );
-  const { address } = useWalletContext();
-  const LoanIcon = getNativeIcon(batch.loan.displaySymbol);
-  const isFocused = useIsFocused();
+
+  const { minNextBidInToken, totalCollateralsValueInUSD, minNextBidInUSD } =
+    useAuctionBidValue(batch, vault.liquidationPenalty);
+
   const {
-    bottomSheetRef,
-    containerRef,
-    dismissModal,
-    expandModal,
-    isModalDisplayed,
-    bottomSheetScreen,
-    setBottomSheetScreen,
-  } = useBottomSheet();
-  const { auctions } = useSelector((state: RootState) => state.auctions);
+    timeRemaining,
+    blocksRemaining,
+    normalizedBlocks,
+    timeRemainingThemedColor,
+  } = useAuctionTime(vault.liquidationHeight, blockCount);
+
+  const isFocused = useIsFocused();
+  const { bottomSheetRef, containerRef, isModalDisplayed, bottomSheetScreen } =
+    useBottomSheet();
+
+  const { getTokenPrice } = useTokenPrice();
 
   useEffect(() => {
     if (isFocused) {
       dispatch(fetchAuctions({ client }));
+      dispatch(
+        fetchBidHistory({
+          vaultId: vault.vaultId,
+          liquidationHeight: vault.liquidationHeight,
+          batchIndex: batch.index,
+          client: client,
+          size: 200,
+        })
+      );
+    } else {
+      dispatch(storeAuctions.actions.resetBidHistory());
```

Comment:
Should remove this line because, this resetting action has a conflict with fetching bid history action in bid history screen, this causes bid history item is resetting then appearing during every block count